projects/plugins/jetpack/extensions/blocks/donations/amount/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { button as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+
+const name = 'donations-amount';
+const BUTTON_BEHAVIOUR_TITLE = __( 'Amount selector', 'jetpack' );
+const INPUT_BEHAVIOUR_TITLE = __( 'Custom amount field', 'jetpack' );
+
+const settings = {
+	title: __( 'Donations amount button', 'jetpack' ),
+	description: __( 'Button that determines a fixed donation amount', 'jetpack' ),
+	category: 'earn',
+	attributes: {
+		baseAmountMultiplier: {
+			type: 'string',
+		},
+		label: {
+			type: 'string',
+		},
+		currency: {
+			type: 'string',
+		},
+		amount: {
+			type: 'string',
+		},
+		disabled: {
+			type: 'boolean',
+		},
+	},
+	__experimentalLabel: ( { disabled } ) =>
+		disabled ? INPUT_BEHAVIOUR_TITLE : BUTTON_BEHAVIOUR_TITLE,
+	icon,
+	supports: {
```

Comment:
All the supports are in place but they have no effect on the actual block.

Reply:
I think the problem is with `__experimentalSkipSerialization` in the block definition.
With that flag, style attributes won't be applied to the actual HTML element.

If you remove all the skips in the block definition and add `useBlockProps` to the block's Edit, the styles magically work. 🙂 

```js
import { RichText, useBlockProps } from '@wordpress/block-editor';

<RichText
  { /* all the existing props */ }
  { ...useBlockProps() }
/>
```

Reply:
It magically worked! I had to update the save.js to save the blockProps and remove some logic from the amount.php file. Thanks for the insight!